### DISCUSSION TITLE
Don't show hidden Mac .files

### DISCUSSION THREAD

**Discussion by kowalsk** [phcode-dev/phoenix#691] [Ideas]
On a Mac computer (or even just a USB drive that was used on a Mac) I can see loads of files that are normally hidden by the OS. Brackets doesn't show them, but Phoenix unfortunately does. Can these be hidden either by default or with some easy setting?

**Comment by abose** [comment]
@kowalsk , Can you provide some sample file names that show up in Phoenix but not in brackets so that I can look at the issue?
